dags/utils/deploy.py

```diff
@@ -0,0 +1,16 @@
+
+def get_artifact_url(slug, branch=None, tag=None,
+                     region="{{ task.__class__.region }}",
+                     bucket="{{ task.__class__.artifacts_bucket }}",
+                     owner_slug="{{ test.__class__.mozilla_slug }}",
```

Comment:
it looks odd that this is test instead of task, but it was like that where it was copied from so 🤷‍♂️